package.json

```diff
@@ -65,6 +65,7 @@
     "@typescript-eslint/parser": "3.6.0",
     "babel-eslint": "10.1.0",
     "babel-loader": "8.1.0",
+    "cross-env": "^7.0.2",
```

Comment:
Good find!

Reply:
Well, good try 😢

Reply:
It fixed one problem, but another one cropped up 🙃